src/components/Personal-Zone/Searcher/FavoriteJobs/FavoriteJobsStyle.ts

```diff
@@ -0,0 +1,39 @@
+import { makeStyles, createStyles } from '@material-ui/core/styles';
+import {  THEME_COLOR, THEME_HOVER_COLOR } from '../../../../assets/projectJSS/Colors';
+
+const useStyles = makeStyles((theme) =>
+    createStyles({
+        root: {
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            padding: '2vh 33vw',
+            lineHeight: 2
+        },
+        content: {
+            display: 'flex',
+            flexDirection: 'column',
+            textAlign: 'center',
+            lineHeight: 1.4,
+            fontSize: '16pt',
+            fontWeight: 500,
+            color: '#686868',
+            marginBottom: '15px'
```

Comment:
Are you sure you don't want to use `vh` here? (e.g if the screen is very large, 15px might not be enough)